### ISSUE TITLE
Setting up Multisite

### ISSUE THREAD

**Issue by amireds** [PhilippHeuer/wordpress-heroku#25]
### Description of your issue
How do activate wordpress multisite. I am say, weldone for this solution

### Steps to Reproduce
...

### Other Information 
Git Commit/Version:

**Comment by PhilippHeuer** [comment]
It should work in theory, but i never tried it myself.

You need to set the following values in your `.env` to enable multisite:
```
WP_ALLOW_MULTISITE=true
WP_MULTISITE_MAIN_DOMAIN=example.com
```

**Comment by amireds** [comment]
Yeah, Figured it out later on. Thanks @PhilippHeuer.
Something else have been battling with....

I set up the multisite as a sub-domain. But on creating a new site, The url keeps taking me to a 404 page. I think I have to set some form of CName or something... Any Cues??

**Comment by PhilippHeuer** [comment]
We would need some adjustments in the nginx configuration for the subdomains, but i don't think that heroku supports subdomains in the first place?
https://github.com/PhilippHeuer/wordpress-heroku/blob/master/config/heroku/nginx.conf

Can you give an example how the url would look like, do you use a custom domain?

**Comment by amireds** [comment]
http://dsa.applyed.herokuapp.com

That's an example of the sub-domain I got. But on accessing it, it shows a 404 heroku page.

**Comment by PhilippHeuer** [comment]
I don't think that herokuapp subdomains support subdomains of their own, if you want to use multisite you need to use a custom domain and create a wildcard dns entry pointing to your heroku app.

I tried to search on the net for subsubdomains suppport natively on heroku, but i haven't found anything except the answer to use custom domains.

**Comment by amireds** [comment]
Hello @PhilippHeuer Having decided on taking to your advice, am having issues mapping my domain to the heroku app. And creating the wildcard dns entry. Please can you kindly suggest a link to a tutorial that can help me through?
